modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher/src/Runtime/StandaloneCliOptions.cs

```diff
@@ -192,7 +192,9 @@ public class StandaloneCliOptions : Dictionary<string, string>, IAgentConfigProv
                     { "it|issuercertstoretype=", "Legacy - do not use.", b => {legacyOptions.Add("it|issuercertstoretype"); } },
                     { $"bs|batchsize=|{StandaloneCliConfigKeys.BatchSize}=", "The size of message batching buffer.",
                         (int i) => this[StandaloneCliConfigKeys.BatchSize] = i.ToString() },
-                    { $"si|iothubsendinterval=|{StandaloneCliConfigKeys.BatchTriggerInterval}=", "The trigger batching interval in seconds.",
+                    { $"bi|batchtriggerinterval=|{StandaloneCliConfigKeys.BatchTriggerInterval}=", "The trigger batching interval in milliseconds.",
```

Comment:
I think the real problem here is the bug that BatchTriggerInterval is not able to take a TimeSpan as argument's value as described in our documentation. I would fix this bug rather than adding a new argument that uses integer milliseconds. the milliseconds issue would be solved by passing something like {00:00:00.100} then.